demux.c

```diff
@@ -80,6 +80,8 @@ typedef struct ts_pid_t
 
     int i_pes_status; /* pes + unscrambled */
     struct ev_timer timeout_watcher;
+
+    uint16_t i_sid;
```

Comment:
Please add a comment saying that this value is only used for statistics. As you know a single PID may be in serveral services.